### ISSUE TITLE
Invitation to work on OpenAgar

### ISSUE THREAD

**Issue by Andrews54757** [JaraLowell/OgarServ#159]
@JaraLowell , Your modifications and edits are very interesting. Even I, sometimes looked at your code for reference. Currently, I am creating a new software called OpenAgar. I am inviting you to work on it with us. The thing that is a pain in OpenAgar though is that it's console is ugly. We need you to make it better. I am also notifying others of the "Big Three" (Ogar, MultiOgar,OgarUL) for help. If you decline, I will be sad, yet I will yield. Please join us. OpenAgar would be great if you joined.

Andrews54757
AJS Development

**Comment by JaraLowell** [comment]
Am actually gone up OgarServ v 1.7 this week if all goes well; it has many of the updates MultiOgar has yet also a few fixes that MultiOgar needed. Also we removed the silly ssl/bind to ip shizle Multi uses as it just a huge memory eater, something we adressed already in a few versions back yet people do not seem to listen and push a whole game server in a http server.. yep... Any ways keep your eyes on our updates, am running some final tests on the different game modes. 

However the 1st release of 1.7 will not have MySQL support yet i will re add that in the comming weeks to come as well as some fixes.
